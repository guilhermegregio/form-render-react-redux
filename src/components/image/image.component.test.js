import React from 'react';
import ReactDOM from 'react-dom';

import Image from './image.component';

it('default render image component', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Image name="Image" className="image" />, div);

    expect(div).toMatchSnapshot();
});
