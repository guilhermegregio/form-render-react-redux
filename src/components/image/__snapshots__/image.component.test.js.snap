// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default render image component 1`] = `
<div>
  <div
    data-reactroot=""
    class="image-component group image"
  >
    <img
      name="Image"
      alt="Image"
    />
  </div>
</div>
`;
